assets/js/util/api.js

```diff
@@ -0,0 +1,50 @@
+/**
+ * Track API errors.
```

Comment:
A bit of a nitpick but this module is not specific to errors, it just happens to only have a single utility so far 😄 

```suggestion
 * API utility functions.
```